rpcs3/Emu/System.cpp

```diff
@@ -83,6 +83,35 @@ void Emulator::SetPath(const std::string& path, const std::string& elf_path)
 	m_elf_path = elf_path;
 }
 
+void Emulator::SetConfigFile(const std::string &config_file)
+{
+	m_config_file = config_file;
+}
+
+void Emulator::LoadConfigFile()
+{
+	if (!m_config_file.empty())
+	{
+		fs::file config(m_config_file, fs::read + fs::create);
+
+		// If given game config file is empty, load the global configuration
+		if (config.size() == 0)
```

Comment:
Creating blank files seems a bit strange.. wouldn't it be better to not create blank files and just check if the file exists?